PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+- [@FredHappyface], who recently joined the administration team.
+
+**Final Say:** The three admins collectively have the final say on all
+matters related to the overall direction, policies, and major decisions
+concerning the project. This ensures that the project remains true to
+its core values and goals while also allowing for flexibility and
+community input.
+
+Project-wide decisions are typically made through consensus among the
+admins. In cases where consensus cannot be reached, a majority vote
+among the admins will determine the final decision.
+
+### Unresponsive Maintainers
+
+If a template maintainer does not respond to an issue or pull request
+(PR) within a **reasonable timeframe** (typically **3 weeks**), the
+admins may step in to make a decision on behalf of the maintainer. This
+ensures that the project continues to move forward and that
+contributions are not stalled unnecessarily.
+
+The process in such cases will be as follows:
+
+1. **Initial Attempt:** Contributors should first try to contact the
+   maintainer via the issue or PR thread.
+2. **Escalation:** If there is no response after **3 weeks**, the issue
+   or PR can be escalated by tagging the admins in a comment;
+   @tinted-theming/admins to tag all admins.
+3. **Admin Decision:** Admins will review the situation and may:
+   - Reach out to the maintainer through other means.
+   - Make a decision on the issue or PR if the maintainer remains
+     unresponsive.
+   - Appoint a temporary or new maintainer if the original maintainer is
+     no longer active.
+
+### Unresponsive Admins
+
+In cases where the admins are unresponsive (e.g., no response for **4
+weeks** on an escalated issue or PR), the community can take the
+following steps:
+
+1. **Community Discussion:** Open a discussion issue or use the
+   project's designated communication channel to gather input from other
+   contributors and maintainers.
+2. **Community Consensus:** If a clear consensus is reached within the
+   community (with input from other maintainers), the decision can be
+   acted upon by those involved in the discussion.
+3. **Request Admin Intervention:** If possible, attempt to contact
+   admins through other channels (email, social media, etc.) to notify
+   them of the situation.
+4. **Temporary Action:** In the absence of admin response, maintainers
+   or active contributors may make temporary decisions, such as merging
+   a critical PR, until the admins return.
+
+**Note**: If an admin remains unresponsive, the other admins will
+continue to carry out their responsibilities and may make decisions on
+behalf of the project as necessary. If all admins become unresponsive,
+the community should follow the steps outlined above.
+
+**Note**: If maintainers or admins expect to be unavailable for an
+extended period, they are encouraged to notify the community in advance,
+if possible, to arrange for temporary coverage or decision-making
+processes.
+
+### Succession Planning
```

Comment:
Do we want a section around "Retiring" or "Stepping Down"?

Reply:
I've added a new commit with this included: https://github.com/tinted-theming/home/pull/101/commits/d051603ef7a68343b8ac36eda463c963404bdc37